src/popover/__tests__/popover.test.tsx

```diff
@@ -308,15 +311,39 @@ describe('ARIA labels', () => {
     expect(wrapper.findBody()!.getElement()).not.toHaveAttribute('role', 'dialog');
   });
 
-  it('accessibility validation basic popover', async () => {
-    const wrapper = renderPopover({
-      children: 'Trigger',
-      content: 'Popover',
-      dismissButton: false,
-      renderWithPortal: true,
+  describe('live region', () => {
+    it('do not render live region when closed', () => {
+      renderPopover({
+        children: 'Trigger',
+        content: 'Popover',
+        dismissButton: false,
+        renderWithPortal: true,
+      });
+      expect(createWrapper().findLiveRegion()).toBeNull();
+    });
+
+    it('render live region when opened and dismissButton = false', () => {
+      const wrapper = renderPopover({
+        children: 'Trigger',
+        content: 'Popover',
+        dismissButton: false,
+        renderWithPortal: true,
+      });
+      wrapper.findTrigger().click();
+      const liveRegion = createWrapper().findLiveRegion()!.getElement()!;
+      expect(liveRegion).toHaveTextContent('Popover');
+    });
+
+    it('do not render live region when opened and dismissButton = true', () => {
+      const wrapper = renderPopover({
+        children: 'Trigger',
+        content: 'Popover',
+        dismissButton: true,
+        renderWithPortal: true,
+      });
+      wrapper.findTrigger().click();
+      expect(createWrapper().findLiveRegion()).toBeNull();
     });
-    wrapper.findTrigger().click();
-    await expect(document.body).toValidateA11y();
```

Comment:
Would it be useful to still call the `toValidateA11y` line to ensure other a11y checks?